.github/workflows/dapr.yml

```diff
@@ -173,3 +190,71 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
+      - name: Package Helm chart
+        env:
+              HELM_CHARTS_DIR: charts/dapr
+        run: |
+          if [ "$LATEST_RELEASE" = "true" ]; then
+            mkdir -p ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+            helm package ${{ env.HELM_CHARTS_DIR }} --destination ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+          fi
+      - name: Upload Helm charts package to artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: dapr_helm_charts_package
+          path: ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+  helmpublish:
+    name: Publish helm charts to Helm github pages repo
+    needs: publish
+    if: github.event_name != 'pull_request'
+    env:
+      ARTIFACT_DIR: ./release
+      DAPR_VERSION_ARTIFACT: dapr_version
+      DAPR_RELEASE_TAG_ARTIFACT: dapr_release
+      HELM_PACKAGE_DIR: helm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Helm charts directory
+        run: |
+          mkdir -p ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+      - name: download artifacts - dapr_helm_charts_package
+        uses: actions/download-artifact@master
+        with:
+          name: dapr_helm_charts_package
+          path: ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+      - name: download artifacts - dapr_version
+        uses: actions/download-artifact@master
+        with:
+          name: ${{ env.DAPR_VERSION_ARTIFACT }}
```

Comment:
this name will always be dapr_release can it be more meaningful

Reply:
The name of the artifact is dapr_version. The value will be the release number which will be used in the commit comment